docs/troubleshooting.md

```diff
@@ -0,0 +1,202 @@
+
+```shell
+$ kubectl describe ingress INGRESS-NAME
+```
+
+You should see an output message providing some information including any ingress issues.
+
+For more troubleshooting tips, see the official k8s docs for [troubleshooting services](https://kubernetes.io/docs/tasks/debug/debug-application/debug-service/).
+
+### Troubleshoot Persistence Volumes
+
+The next common problem to occur is Persistent Volumes failing to provision. To check the status of your persistent volumes, run *kubectl get pvc*. If the output message shows that your PVC status is pending, this may be because your cluster does not support dynamic provisioning (such as a bare metal cluster). In this case, the pod is unable to start because the cluster is unable to fulfil the request to attach the persistent volume to your pod.
+
+To get detailed information around the PVC, replacing `PVC-NAME` with the PVC you want to investigate, run:
+
+```shell
+$ kubectl describe PVC-NAME
+```
+
+To fix this, you must manually configure some persistent volumes or set up a *StorageClass* resource and provisioner for dynamic volume provisioning, such as the [NFS provisioner](https://github.com/kubernetes-incubator/external-storage/tree/master/nfs). [Learn more about dynamic provisioning and storage classes](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/).
+
+You can also check out the official K8s documentation for a list of supported [volumes](https://kubernetes.io/docs/concepts/storage/volumes/).
+
+### Troubleshoot Helm Charts
+
+The following are the most common issues users face when installing Helm charts:
+
+* Connection time out when installing a chart
+* Credential errors due to upgrades and existing Persistence Volumes (PVs) from previous releases
+
+When a timeout occurs installing a chart, the error will look similar to this:
+
+```shell
+Error: INSTALLATION FAILED: Kubernetes cluster unreachable: Get "https://IP-ADDRESS/version": dial tcp IP-ADDRESS connect: connection timed out
+```
+
+To fix, wait a few minutes then try again. If the problem persists, ensure you are connected to the cluster by running a  command such as *kubectl get pods* or increase the `--timeout` field when installing the chart.
+
+Another common issue is credential related errors, caused by a [known issue](https://github.com/bitnami/charts/issues/2061) with PostgreSQL authentication failing after re-deployment. This occurs when you delete the deployed Helm chart while keeping the existing persistent volumes as they are not removed by default. When re-deploying the chart using the same release name, this causes the secret generating the credentials to go out-of-sync with the password being persisted.
+
+To prevent, ensure the underlying PVC has also been removed when uninstalling the chart or use a different release name. For example, to delete PVCs related to the PostgreSQL deployment, run:
```

Comment:
I think that besides the PVC you also need to delete the PV (at least I had to when I did a uninstall/install cycle with helm.

Reply:
Agreed, updated to have the user delete both